tdrs-backend/tdpservice/parsers/schema_defs/tanf/t7.py

```diff
@@ -0,0 +1,61 @@
+"""Schema for TANF T7 Row."""
+
+from ...util import SchemaManager
+from ...fields import Field, TransformField
+from ...row_schema import RowSchema
+from ...transforms import calendar_quarter_to_rpt_month_year
+from ... import validators
+from tdpservice.search_indexes.models.tanf import TANF_T7
+
+schemas = []
+
+validator_index = 7
+section_ind_index = 7
+stratum_index = 8
+families_index = 10
+for i in range(1, 31):
+    schemas.append(
+      RowSchema(
+          model=TANF_T7,
+          quiet_preparser_errors=i > 1,
+          preparsing_validators=[
+              validators.notEmpty(0, 7),
+              validators.notEmpty(validator_index, validator_index + 24),
+          ],
+          postparsing_validators=[],
+          fields=[
+              Field(item="0", name="RecordType", type='string', startIndex=0, endIndex=2,
+                    required=True, validators=[]),
+              Field(item="3", name='CALENDAR_QUARTER', type='number', startIndex=2, endIndex=7,
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.quarterIsValid()]),
+              TransformField(
+                  transform_func=calendar_quarter_to_rpt_month_year((i - 1) % 3),
+                  item="3A",
+                  name='RPT_MONTH_YEAR',
+                  type='number',
+                  startIndex=2,
+                  endIndex=7,
+                  required=True,
+                  validators=[
+                      validators.dateYearIsLargerThan(1998),
+                      validators.dateMonthIsValid()
+                  ]
+              ),
+              Field(item="4", name='TDRS_SECTION_IND', type='string', startIndex=section_ind_index,
+                    endIndex=section_ind_index + 1, required=True, validators=[validators.oneOf(['1', '2'])]),
+              Field(item="5", name='STRATUM', type='string', startIndex=stratum_index,
+                    endIndex=stratum_index + 2, required=True, validators=[validators.isInStringRange(1, 99)]),
+              Field(item="6A", name='FAMILIES_MONTH', type='number', startIndex=families_index,
```

Comment:
@jtimpe this is a bit of a nit, but i wonder if users will be confused by the 6A references. in the feedback report. 6A comes up for 6A, 6B, 6C family counts in the feedback report. what do you think @reitermb ? if its an issue, we can wrap it up in #2714 

![1111p6](https://github.com/raft-tech/TANF-app/assets/63075587/c617c9f0-3a58-45bd-933c-efaaebed5da7)

Reply:
this made sense to me so i went ahead and made the change

Reply:
thanks! i need to make a note of this for section 3 #2713

Reply:
![1111p8](https://github.com/raft-tech/TANF-app/assets/63075587/9a641bd7-6bad-4682-93e3-b3b3040934ea)